golint/golint.go

```diff
@@ -22,6 +22,7 @@ import (
 var (
 	minConfidence = flag.Float64("min_confidence", 0.8, "minimum confidence of a problem to print it")
 	setExitStatus = flag.Bool("set_exit_status", false, "set exit status to 1 if any issues are found")
+	checkVendor   = flag.Bool("vendor", false, "include packages in vendor dirs when expanding ...")
```

Comment:
I don't agree with this. `cmd/go` doesn't have a flag, and there is no reason why `golint` would need one.

Reply:
Right. As discussed in #303 we should mimic go fmt behavior (no flag).